lib/widgets/compose_box.dart

```diff
@@ -176,23 +186,50 @@ class _StreamContentInputState extends State<_StreamContentInput> {
     });
   }
 
+  _changed() {
+    final newAutocompleteIntent = widget.controller.autocompleteIntent();
+    if (newAutocompleteIntent != null) {
+      if (_mentionAutocompleteView == null) {
+        _mentionAutocompleteView = MentionAutocompleteView.init(
+          store: widget.store, narrow: widget.narrow);
```

Comment:
compose [nfc]: Have _StreamComposeBoxState provide store to children
    
    In particular, _StreamContentInput, so it can initialize
    [MentionAutocompleteView]s soon. But might as well pass it along to
    various other children so they don't have to get it themselves.

Hmm, this doesn't feel idiomatic to me. I think the usual thing would be to just have each widget get the store for itself, given that we've already put it on an `InheritedWidget`.

At this spot in particular, a key fact is that `State` has a `context` getter of its own. So we can write:

```suggestion
        final store = PerAccountStoreWidget.of(context);
        _mentionAutocompleteView = MentionAutocompleteView.init(
          store: store, narrow: widget.narrow);
```